Sources/Build/ManifestBuilder.swift

```diff
@@ -135,13 +135,27 @@ extension LLBuildManifestBuilder {
 
         var outputs: [Node] = []
 
+        let infoPlistDestination = RelativePath("Info.plist")
+        var resourcesContainsInfoPlist = false
+
         // Create a copy command for each resource file.
         for resource in target.target.underlyingTarget.resources {
-            let destination = bundlePath.appending(component: resource.path.basename)
+            if resource.destination == infoPlistDestination {
```

Comment:
Hmm, it seems dangerous to even allow Info.plist in resources as it'll not become a resource but a configuration file for the generated bundle. Let me check with some other folks on what we should do here.

Reply:
Done.